Apidaze.SDK.Tests.Integration/GetSipUsers/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Apidaze.SDK;
+using Apidaze.SDK.Base;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+
+namespace GetSipUsersExample
+{
+    class Program
+    {
+        /// <summary>
+        /// Defines the entry point of the application.
+        /// </summary>
+        /// <param name="args">The arguments.</param>
```

Comment:
args are the arguments? No way Sherlock :) But what kind of arguments do you expect? :)